admin-part/src/main/kotlin/storage/Build.kt

```diff
@@ -42,6 +44,13 @@ class StoredBuildTests(
     val data: BuildTests,
 ) : java.io.Serializable
 
+@Serializable
+internal class StoredClassBytes(
+    @Id val agentId: String,
+    @StreamSerialization(SerializationType.FST, CompressType.ZSTD)
+    val classBytes: Map<String, ByteArray>,
+) : java.io.Serializable
```

Comment:
Why we use only agent id instead of composite id like this ?
`@Serializable
data class CompositeAgentId(
     val agentId: String,
     val buildVersion: String,
    )`

Reply:
I mean, will there be any errors if we connect a new build?